### PR TITLE
MIG-1418: MTC 1.8.0 release notes

### PR DESCRIPTION
**Pull request by anarnold97**
[MIG-1453](https://issues.redhat.com/browse/MIG-1453)

Version(s):

Enterprise 4.11 → branch/enterprise-4.11
Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14

Issue:

https://issues.redhat.com/browse/MIG-1418

## Link to docs preview:

[MTC 1.8.0 Technical Changes](https://65437--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/mtc-release-notes#technical-changes-1-8_mtc-release-notes)

QE review:
- [ ] QE has approved this change.
For approvals, see:
* https://github.com/openshift/openshift-docs/pull/64678
* https://github.com/openshift/openshift-docs/pull/65453
* https://issues.redhat.com/browse/MIG-1453

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
For approvals, see - https://github.com/openshift/openshift-docs/pull/64678

**Comment by tmalove** [comment]
@anarnold97  Per the comment in this MTC PR: https://github.com/openshift/openshift-docs/pull/65660, we are not updating the 3.11 or 4.10 releases unless there is a critical situation. Let me know if this is the case and I will engage the OPC DPM. Thanks!

**Review by stillalearner — APPROVED** [review]
lgtm

**Comment by tmalove** [comment]
> > @anarnold97 Per the comment in this MTC PR: #65660, we are not updating the 3.11 or 4.10 releases unless there is a critical situation. Let me know if this is the case and I will engage the OPC DPM. Thanks!
> 
> @tmalove - thanks very much for checking.
> 
> I checked with the PM for MTC, the docs are not actually in the OpenShift 3.x docs. So please, can we just focus on 4.11.
> 
> It was a huge panic last night... but the PM does not officially care. PHEW!
> 
> I have updated the versions accordingly

Sounds good @anarnold97 !!

**Comment by anarnold97** [comment]
@tmalove - I am sorry to bother you but this is on GA tomorrow. Please can you give me an estimate of how long the review will take?
Thanks

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+In previous releases, on an Azure cluster, when backing up to Azure storage, the migration failed at the `Backup` stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2238974[*BZ#2238974*])
+
+[id="known-issues-1-8_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Old Restic pods are not getting removed on upgrading MTC 1.7.x -> 1.8.x
+
+In this release, on upgrading the MTC operator from 1.7.x to 1.8.x, the old Restic pods are not being removed. Hence, after the upgrade, both Restic and node-agent pods are visible in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2236829[*BZ#2236829*])
```
Comment (tmalove) [inline modules/migration-mtc-release-notes-1-8.adoc:38]:
```suggestion
In this release, on upgrading the MTC operator from 1.7.x to 1.8.x, the old Restic pods are not being removed. Therefore after the upgrade, both Restic and node-agent pods are visible in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2236829[*BZ#2236829*])
```

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+In this release, after enabling `Require SSL verification` and adding content to the CA bundle file for an MCG NooBaa bucket in MigStorage, the connection fails as expected. However, when reverting these changes by removing the CA bundle content and clearing `Require SSL verification`, the connection still fails. The issue is only resolved by deleting and re-adding the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2240052[*BZ#2240052*])
+
+
+[id="technical-changes-1-8_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+* Migration from {product-title} 3 to {product-title} 4 requires a legacy {mtc-full} ({mtc-short}) Operator and {mtc-short} 1.7.x.
```
Comment (tmalove) [inline modules/migration-mtc-release-notes-1-8.adoc:54]:
```suggestion
* Migration from {product-title} 3 to {product-title} 4 requires a traditional {mtc-full} ({mtc-short}) Operator and {mtc-short} 1.7.x.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* Migration from {mtc-short} 1.7.x to {mtc-short} 1.8.x is not supported.
+* You must use {mtc-short} 1.7.x to migrate anything with a source of {product-title} 4.9 or earlier.
+** {mtc-short} 1.7.x must be used on both source and destination.
+* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x may be used. However, but it must be the same MTC 1.Y.z on both source & destination.
```
Comment (tmalove) [inline modules/migration-mtc-release-notes-1-8.adoc:58]:
```suggestion
* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x might be used. However, but it must be the same MTC 1.Y.z on both source & destination.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.8.x is unsupported.
+** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.7.x is unsupported.
+** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.7.x is supported.
+** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.8.x is supported
+* MTC 1.8.x by default installs OADP 1.2.x.
+* Upgrading from {mtc-short} 1.7.x to {mtc-short} 1.8.0, requires manually changing the OADP channel to 1.2. If this is not done, the upgrade of the operator will fail.
```
Comment (tmalove) [inline modules/migration-mtc-release-notes-1-8.adoc:64]:
```suggestion
* Upgrading from {mtc-short} 1.7.x to {mtc-short} 1.8.0, requires manually changing the OADP channel to 1.2. If this is not done, the upgrade of the operator fails.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice

**Review by tmalove — APPROVED** [review]

**Comment by anarnold97** [comment]
Please can I flag that GA is tomorrow and we have waited for this to come peer-review for a long time
Thanks

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+In previous releases, on an Azure cluster, when backing up to Azure storage, the migration failed at the `Backup` stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2238974[*BZ#2238974*])
+
+[id="known-issues-1-8_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Old Restic pods are not getting removed on upgrading MTC 1.7.x -> 1.8.x
+
+In this release, on upgrading the MTC operator from 1.7.x to 1.8.x, the old Restic pods are not being removed. Therefore after the upgrade, both Restic and node-agent pods are visible in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2236829[*BZ#2236829*])
```
Comment (adellape) [inline modules/migration-mtc-release-notes-1-8.adoc:38]:
```suggestion
In this release, on upgrading the MTC Operator from 1.7.x to 1.8.x, the old Restic pods are not being removed. Therefore after the upgrade, both Restic and node-agent pods are visible in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2236829[*BZ#2236829*])
```

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+* Migration from {mtc-short} 1.7.x to {mtc-short} 1.8.x is not supported.
+* You must use {mtc-short} 1.7.x to migrate anything with a source of {product-title} 4.9 or earlier.
+** {mtc-short} 1.7.x must be used on both source and destination.
+* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x might be used. However, but it must be the same MTC 1.Y.z on both source & destination.
+** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.8.x is unsupported.
+** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.7.x is unsupported.
+** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.7.x is supported.
+** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.8.x is supported
+* MTC 1.8.x by default installs OADP 1.2.x.
+* Upgrading from {mtc-short} 1.7.x to {mtc-short} 1.8.0, requires manually changing the OADP channel to 1.2. If this is not done, the upgrade of the operator fails.
```
Comment (adellape) [inline modules/migration-mtc-release-notes-1-8.adoc:64]:
```suggestion
* Upgrading from {mtc-short} 1.7.x to {mtc-short} 1.8.0, requires manually changing the OADP channel to 1.2. If this is not done, the upgrade of the Operator fails.
```

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+* Migration from {product-title} 3 to {product-title} 4 requires a legacy {mtc-full} ({mtc-short}) Operator and {mtc-short} 1.7.x.
+* Migration from {mtc-short} 1.7.x to {mtc-short} 1.8.x is not supported.
+* You must use {mtc-short} 1.7.x to migrate anything with a source of {product-title} 4.9 or earlier.
+** {mtc-short} 1.7.x must be used on both source and destination.
+* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x might be used. However, but it must be the same MTC 1.Y.z on both source & destination.
+** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.8.x is unsupported.
+** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.7.x is unsupported.
+** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.7.x is supported.
+** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.8.x is supported
```
Comment (adellape) [inline modules/migration-mtc-release-notes-1-8.adoc:62]:
```suggestion
** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.8.x is supported.
```

File: modules/migration-mtc-release-notes-1-8.adoc [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+[id="technical-changes-1-8_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+* Migration from {product-title} 3 to {product-title} 4 requires a legacy {mtc-full} ({mtc-short}) Operator and {mtc-short} 1.7.x.
+* Migration from {mtc-short} 1.7.x to {mtc-short} 1.8.x is not supported.
+* You must use {mtc-short} 1.7.x to migrate anything with a source of {product-title} 4.9 or earlier.
+** {mtc-short} 1.7.x must be used on both source and destination.
+* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x might be used. However, but it must be the same MTC 1.Y.z on both source & destination.
```
Comment (adellape) [inline modules/migration-mtc-release-notes-1-8.adoc:58]:
```suggestion
* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x might be used. However, but it must be the same MTC 1.Y.z on both source and destination.
```

**Review by adellape — COMMENTED** [review]
Not blocking merge for these, but potential follow-up fixes if you like.

**Comment by anarnold97** [comment]
Let me fix and squash now

**Pull request merged**